### ISSUE TITLE
test problem

### ISSUE THREAD

**Issue by dangvansam** [xxbb1234021/speech_recognition#25]
i haven train a model (50 epochs, datatrain 11000 wave files with text for each file, batchsize 16, learning rate = 0.001)
when training: 
train_err:  0.028019862
trans_array_to_text_ch len: 72
words [' ', '4', ':', 'A', 'B', 'C', 'D', 'E', 'G', 'H', 'I', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'X', 'Y', 'À', 'Á', 'Â', 'Ã', 'È', 'É', 'Ê', 'Ì', 'Í', 'Ò', 'Ó', 'Ô', 'Õ', 'Ù', 'Ú', 'Ý', 'Ă', 'Đ', 'Ĩ', 'Ũ', 'Ơ', 'Ư', 'Ạ', 'Ả', 'Ấ', 'Ầ', 'Ẩ', 'Ẫ', 'Ậ', 'Ắ', 'Ằ', 'Ẳ', 'Ẵ', 'Ặ', 'Ẹ', 'Ẻ', 'Ẽ', 'Ế', 'Ề', 'Ể', 'Ễ', 'Ệ', 'Ỉ', 'Ị', 'Ọ', 'Ỏ', 'Ố', 'Ồ', 'Ổ', 'Ỗ', 'Ộ', 'Ớ', 'Ờ', 'Ở', 'Ỡ', 'Ợ', 'Ụ', 'Ủ', 'Ứ', 'Ừ', 'Ử', 'Ữ', 'Ự', 'Ỳ', 'Ỵ', 'Ỷ', 'Ỹ'] value [20 18 15 14  8  0 20  9 77 10  0  8 10  3 14  0 42 50 21  0 14  8 46 77
 10  0 13 76 10  0 42 62 14  0  5  9 46  3  0  4 10 62 20  0  5 26  5  9
  0 12 25 13  0 22 10 66  5  0  9 10 66 21  0 17 21 48 -1 -1 -1 -1 -1 -1] results TRONG THỜI GIAN ĐẦU NGƯỜI MỚI ĐẾN CHƯA BIẾT CÁCH LÀM VIỆC HIỆU QUẢỸỸỸỸỸỸ
orig: TRONG THỜI GIAN ĐẦU NGƯỜI MỚI ĐẾN CHƯA BIẾT CÁCH LÀM VIỆC HIỆU QUẢ
decoded_str:  TRONG THỜI GIAN ĐẦU NGƯỜI MỚI ĐẾN CHƯA BIẾT CÁCH LÀM VIỆC HIỆU QUẢỸỸỸỸỸỸ
----------------------------------------------------------------------------------------------
but when train done i test model text is not recognition
wav_files[0]:  data_test\VIVOSDEV01_R117.wav
Testing......
trans_array_to_text_ch len: 65
words [' ', 'A', 'B', 'C', 'D', 'E', 'G', 'H', 'I', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'X', 'Y', 'À', 'Á', 'Â', 'Ã', 'È', 'É', 'Ê', 'Ì', 'Í', 'Ò', 'Ó', 'Ô', 'Õ', 'Ù', 'Ú', 'Ý', 'Ă', 'Đ', 'Ĩ', 'Ũ', 'Ơ', 'Ư', 'Ạ', 'Ả', 'Ấ', 'Ầ', 'Ẩ', 'Ẫ', 'Ậ', 'Ắ', 'Ằ', 'Ẳ', 'Ặ', 'Ẹ', 'Ẻ', 'Ẽ', 'Ế', 'Ề', 'Ể', 'Ễ', 'Ệ', 'Ỉ', 'Ị', 'Ọ', 'Ỏ', 'Ố', 'Ồ', 'Ổ', 'Ỗ', 'Ộ', 'Ớ', 'Ờ', 'Ở', 'Ỡ', 'Ợ', 'Ụ', 'Ủ', 'Ứ', 'Ừ', 'Ử', 'Ữ', 'Ự', 'Ỳ', 'Ỷ', 'Ỹ'] value [21 67 21 24 48 24 48 24 48 24  4 24  4 24  4 16  4 24 21 16 21 24 21  4
 21  4 21  4 24  4 21 48 21  4 24 16 21 16 21 16 21 16  4 87 21 16 21 16
 24 21 24  4 24  4 16 45 16 24  4 16  4 16  4 46 16] results XỎXÁẦÁẦÁẦÁDÁDÁDRDÁXRXÁXDXDXDÁDXẦXDÁRXRXRXRDỸXRXRÁXÁDÁDRẠRÁDRDRDẢR
orig: VIỆC MỘT QUYỂN TIỂU THUYẾT NHƯ THẾ
decoded_str:  XỎXÁẦÁẦÁẦÁDÁDÁDRDÁXRXÁXDXDXDÁDXẦXDÁRXRXRXRDỸXRXRÁXÁDÁDRẠRÁDRDRDẢR
